### PR TITLE
Commands to show list of the largest objects

### PR DESCRIPTION
**Pull request by avallac**
Hi

Usually, to find the problem, just look at the list of the largest objects

### CODE REVIEW

**Comment by avallac** [comment]
@BitOne

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (BitOne) [inline .gitignore]: It's a specific configuration linked to your own environment. It should be done inside your global gitignore configuration, not in the configuration of this project ;) https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

File: analyzer/src/BitOne/PhpMemInfo/Analyzer/TopSearcher.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                    } elseif ($i !== 0) {
+                        $top[$i - 1] = $top[$i];
+                    }
+                }
+            }
+        }
+
+        return $top;
+    }
+}
```
Comment (BitOne) [inline analyzer/src/BitOne/PhpMemInfo/Analyzer/TopSearcher.php]: Missing end of line at end of file.

File: analyzer/src/BitOne/PhpMemInfo/Analyzer/TopSearcher.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+     * @var int $limit
+     * @return array
+     */
+    public function createTopChildren($limit)
+    {
+        $top = [];
+        for($i = 0; $i < $limit; $i ++) {
+            $top[] = 0;
+        }
+        foreach ($this->items as $key => $item) {
```
Comment (BitOne) [inline analyzer/src/BitOne/PhpMemInfo/Analyzer/TopSearcher.php]: I would go for something far much simpler in term of algorithm, even if less efficient:
 - create a second array with key as item key and value is the number of children
 - call asort on it
 - call array_slice to get the x top objects with children (using $limit)

**Comment by avallac** [comment]
@BitOne
Do you have a good dataset for check 'size_with_child' ? I want to test my implementation, maybe it will be good.

**Comment by avallac** [comment]
@BitOne ?

**Comment by avallac** [comment]
Hi @BitOne

I made changes.

**Comment by BitOne** [comment]
Thanks @avallac, this looks pretty good!

**Pull request merged**